### DISCUSSION TITLE
Battery not charging enough overnight

### DISCUSSION THREAD

**Discussion by idpandjab** [nathanmarlor/foxess_em#368] [Q&A]
Nathan I set this up Friday last week. Since then it wored ok Monday night but last night it set the min SOC at 17% and didn't put enough into the battery to survive the day. I manually put it on charge at 7:30 this morning. Can you advise if it needs more time to learn or if I may have set it up wrong. I have 3kWp solar. 13.5kW cube batteries with a 5kW AC1 inverter. The solar is a separate system. I probably use 8-10kW a day, so on a day like today where the forecast is ~6kW I need to have sufficient charge in the battery to survive. Thx for any help

Ian
![image](https://github.com/user-attachments/assets/f935c562-fa42-4be3-bf60-7d32cbfb5094)
![image](https://github.com/user-attachments/assets/81c342f5-36bf-4fed-be5f-bdbfc27aea9f)
![image](https://github.com/user-attachments/assets/f47a458b-d027-4a9d-ba12-4915717c2e54)
![image](https://github.com/user-attachments/assets/c610e166-b967-4f12-aad8-0a2e27be9aad)
![image](https://github.com/user-attachments/assets/675d384e-c2f2-4647-a5de-8dc870a785dd)

**Comment by markhaines** [comment]
It obviously depends what tariff you're on, but typically your strategy should be to set the battery to charge to 100% overnight on your cheap time slot, then during the day use solar to power the house and export any surplus to the grid, leaving the battery untouched. The battery just needs to tide you over in the evening as solar generation falls and doesn't cover your house load until your cheap period kicks in (and help out during the day in the winter), rinse and repeat.  You can mess about with charging the battery overnight and then exporting a portion of this as well during 'peak export' hours, but the £ return on this is marginal so I don't bother.

**Reply by idpandjab** [reply]
E-ON next drive. Currently 01:30-08:30 cheap rate. I don't want to export as I don't get paid for it. On a good solar day I can generate 16kW which tops off the battery, excess goes into hot water. But if I get the battery SOC right I don't generate any excess. But like today I only generated 5.7kW so not enough to run the house and keep the battery SOC stable. But the question is why did it sit at 15% last night and not charge the batteries higher?
